src/frontend/e2e/tests/sample.spec.ts

```diff
@@ -30,12 +34,21 @@ test.describe("Samples page tests", () => {
   });
 
   test("Should verify sample data", async ({ page }: { page: Page }) => {
+    await page.goto(process.env.BASEURL+"data/samples" as string);
+
     //wait until data is displayed
     await page.waitForSelector('[data-test-id="table-row"]');
-
-    // assert table is populated with at least one record
-    expect(
-      await page.locator(getByTestID("table-row")).count()
-    ).toBeGreaterThan(0);
+    await expect(page.locator('text="20SCPH11281"')).not.toBeEmpty();
```

Comment:
what are all these magic values? 
how will a developer know if a new test failure is valid or not/when would these change?
won't these pass/fail depending on the environment (for example if we add a bunch of samples to staging)?